rollup.config.ts

```diff
@@ -4,6 +4,8 @@ import jsonPlugin from '@rollup/plugin-json'
 import nodeResolvePlugin from '@rollup/plugin-node-resolve'
 import typescriptPlugin from '@rollup/plugin-typescript'
 import terserPlugin from '@rollup/plugin-terser'
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore Otherwise ts-node reject to run make_selectors_tester.ts
 import dtsPlugin from 'rollup-plugin-dts'
```

Comment:
Error happens because of a `"type": "module"` field in the `package.json` of `rollup-plugin-dts`. I tried different [moduleTypes](https://typestrong.org/ts-node/docs/module-type-overrides/) values, it didn't work.

Reply:
- https://github.com/microsoft/TypeScript/issues/53045
- https://github.com/microsoft/TypeScript/issues/49299

A PR for fixing this on the `rollup-plugin-dts` side: https://github.com/Swatinem/rollup-plugin-dts/pull/212

Reply:
Addressed by https://github.com/fingerprintjs/fingerprintjs/pull/932